.github/workflows/pr.yaml

```
@@ -19,45 +19,34 @@ jobs:
 
       - run: git fetch
       - name: Check that changelog has been updated.
-        run: git diff --exit-code origin/${{ github.base_ref }} -- CHANGELOG.md
-
-  composer-validate:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - run: |
-          docker network create frontend
-      - run: |
-          docker compose run --rm phpfpm composer validate
-
-  composer-normalize:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - run: |
-          docker network create frontend
-          docker compose run --rm phpfpm composer install
-      - run: |
-          docker compose run --rm phpfpm composer normalize --dry-run
+        run: git diff --exit-code origin/${{ github.base_ref }} -- CHANGELOG.md && exit 1 || exit 0
 
   coding-standards:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         script:
+          - validate
+          - normalize --dry-run
           - code-analysis
           - coding-standards-check/php
           - coding-standards-check/twig
     steps:
       - uses: actions/checkout@v4
-      - uses: pnorton5432/setup-task@v1
       - run: |
           docker network create frontend
           docker compose run --rm phpfpm composer install
       - run: |
           docker compose run --rm phpfpm composer ${{ matrix.script }}
 
+  coding-standards-markdown:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          docker run --rm --volume "$PWD:/md" peterdavehello/markdownlint markdownlint --ignore '**/vendor/**' '**.md'
```

```suggestion
          docker run --rm --volume "$PWD:/md" peterdavehello/markdownlint markdownlint  '**.md'
```

duplicate with `[.markdownlintignore](https://github.com/itk-dev/hoeringsportal-getorganized/pull/13/files#diff-b5e89e929d119744a043caea30a5e49d7a9ea25e3253755760d5b6ec9bd24ca0)`